src/main/java/players/rmhc/RMHCPlayer.java

```diff
@@ -21,6 +21,7 @@
     private double avgTimeTaken = 0, acumTimeTaken = 0;
     private int numIters = 0;
     private int fmCalls = 0;
+    private int copyCalls =0;
```

Comment:
This is not part of what we've traditionally used as "FM call budget", specifically it's the number of calls to the FM.next() function. We can add another type of budget to include copies as well, but it should not affect the other settings for comparison to previous experiments.

Reply:
Very fair. I'll separate these two. In passing it migt be worth noting that for all the eight implemented games currently, the copy() method quite a bit more time-consuming then the next() method.

(My main beef with the previous code as actually that in Closed Loop MCTS it incremented the fmCalls budget even though the fm wasn't called! Removing this too naively then led to a potential infinite loop...but I've now added a specific check for that so that we stop if the number of iterations ever exceeds the Forward Model budget)